ros/api/v1/hosts.py

```diff
@@ -477,6 +487,32 @@ def get(self):
 
         all_conditions_count = total_cpu_issues + total_memory_issues + total_io_issues
 
+        current_utc_datetime = datetime.utcnow().replace(tzinfo=pytz.utc)
+
+        # target upload date is yesterday's date
```

Comment:
@saltgen  - This variable is used only once in query while applying filter on `report_date`.
I am fine to keep it as `upload_date`.  I suggested this change because from assignment (`(current_utc_datetime - timedelta(days=1)).date()`) it is confusing why it is being called as `upload_date`.

Reply:
@kgaikwad Kept it as `upload_date` as  that is currently the on wards date we are considering for profile uploads in `current` and `suggested` section, would you like to suggest further changes?